lib/vanagon/platform/windows.rb

```diff
@@ -54,8 +54,50 @@ def generate_packaging_artifacts(workdir, name, binding)
       # @param binding [Binding] binding to use in evaluating the packaging templates
       def generate_msi_packaging_artifacts(workdir, name, binding)
         FileUtils.mkdir_p(File.join(workdir, "wix"))
-        erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/project.wxs.erb"), File.join(workdir, "wix",  "#{name}.wxs"), false, { :binding => binding })
-        erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/project.filter.xslt.erb"), File.join(workdir, "wix", "#{name}.filter.xslt"), false, { :binding => binding })
+        currentwd = Dir.getwd
+
+        # Recurse through the list of files in the resources/windows/wix directory and copy them over to the work directory.
+        # Treat special cases as necessary (e.g. erb files)
+        # Could probably do this as a function except for xslt/project special cases
+        Dir.chdir("resources/windows/wix")
+        Dir.glob("**/*.*").each do |source_file|
+          src_dirname = File.dirname(source_file)
+          src_basename = File.basename(source_file)
+          target_dir = File.join(workdir, "wix", src_dirname)
+          FileUtils.mkdir_p(target_dir)
+
+          case
+          when src_basename.match(/ˆ.*\.erb$/)
+            new_filename = File.basename(src_basename, ".erb")
+            erb_file(source_file, File.join(target_dir, new_filename), false, { :binding => binding })
+          else
+            FileUtils.copy(source_file, File.join(target_dir, src_basename), :verbose => true)
+          end
+        end
```

Comment:
This block is pretty similar to the block 79-97 - should I simply express this as a method which could potentially allow for the xslt/product file to be present in the project specific case.

Another thought would be to test if the destination file exists and then don't copy - this would give precedence to project specific files.